applications/MappingApplication/custom_searching/interface_communicator.cpp

```diff
@@ -75,7 +75,9 @@ InterfaceCommunicator::InterfaceCommunicator(
         "max_num_search_iterations"     : 0,
         "print_bounding_boxes_to_file"  : false,
         "bounding_boxes_file_path"      : "",
-        "echo_level"                    : 0
+        "echo_level"                    : 0,
+        "required_rbf_support_points"  : 0,
+        "use_all_rbf_support_points": true
```

Comment:
why not define "max_support_points`?

Also, this should be settings of the RBF local system and interface info, its not even used here